leapp/utils/report.py

```diff
@@ -42,10 +42,24 @@ def fetch_upgrade_report_messages(context_id):
     return messages
 
 
+IMPORTANCES = {
```

Comment:
Call it SEVERITY_LEVELS. Importance does not go well with Severity.

Reply:
Alright.